rtf_vis_tool/package.json

```diff
@@ -14,7 +14,7 @@
     "react-scripts": "4.0.1",
     "react-three-fiber": "^5.3.12",
     "react-xarrows": "^1.5.2",
-    "three": "^0.127.0"
+    "three": "^0.124.0"
```

Comment:
small dependency version conflict here. Just had to downgrade three to fix it

Reply:
and because I modified the version of the `three` dependency within `package.json`, that also modified the `package-lock.json` file as well (no noteworthy changes though)